en/tools/config-catalog-generator/data/apim.analytics.url_group.toml

```diff
@@ -0,0 +1,11 @@
+#LOADBALANCER CONFIGS
+[[apim.analytics.url_group]]
+analytics_url =["tcp://analytics1:7611","tcp://analytics2:7611"]
```

Comment:
Shall we add a space after the "=" sign in all of the configs in this file?

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/3004